<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNombre">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelApellido">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Apellido:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtApellido">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelApellido_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Celular:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelApellido_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>110</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Correo:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtCelular">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtCorreo">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>110</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelApellido_4">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>40</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Direccion:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelApellido_5">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>110</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Apodo:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDireccion">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>40</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtApodo">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>110</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelApellido_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Cumplea√±os:</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>170</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGuardar">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>170</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Guardar Contacto</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Crear un nuevo contacto</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>170</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>738</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpcion">
    <property name="title">
     <string>Opciones</string>
    </property>
    <addaction name="actionActualizar"/>
    <addaction name="actionEliminar"/>
    <addaction name="actionMostrar_Contactos"/>
   </widget>
   <addaction name="menuOpcion"/>
  </widget>
  <action name="actionLabelActu">
   <property name="text">
    <string>LabelActu</string>
   </property>
  </action>
  <action name="actionActualizar">
   <property name="text">
    <string>Actualizar</string>
   </property>
  </action>
  <action name="actionEliminar">
   <property name="text">
    <string>Eliminar</string>
   </property>
  </action>
  <action name="actionMostrar_Contactos">
   <property name="text">
    <string>Mostrar Contactos</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
